src/requestor/routes/manage.py

```diff
@@ -210,6 +209,9 @@ async def create_request(
 
     if draft_previous_requests:
         # reuse the draft request
+        logger.debug(
+            f"Found a draft request with request_id: {draft_previous_requests[0].request_id}"
```

Comment:
Can we put this log after the `to_dict` part and access the `request_id` as if we are accessing the key in the dict? I know there's probably going to be a very minute difference in performance but it is better nonetheless. Or am I wrong? wdyt?

Reply:
i don't think it makes a difference
generally i like to log what i'm about to do before i do it